tools/pythonpkg/src/pyresult.cpp

```diff
@@ -264,12 +264,57 @@ py::object DuckDBPyResult::FetchArrowTable() {
 	return from_batches_func(batches, schema_obj);
 }
 
+py::str GetTypeToPython(const LogicalType &type) {
+	switch (type.id()) {
+	case LogicalTypeId::BOOLEAN:
+		return py::str("bool");
```

Comment:
I struggled to find any real standard for these values - they seem to largely be user-defined, beyond the explicit ones in the DBAPI2.0 spec

I'm happy to take suggestions for ones that make more sense

Reply:
Perhaps we can align with either the psycopg2 driver, since Postgres ~matches our set of supported types?

Reply:
psycopg2 uses the postgres type OID for the type_code field - does duckdb have an equivalent?

Reply:
DuckDB supports the type `oid` as an alias to `bigint`.

Reply:
Sorry, I misworded, I mean the value of the type_code field is the OID of the column - see the docs: https://www.psycopg.org/docs/extensions.html#psycopg2.extensions.Column.type_code